src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -11019,7 +11025,13 @@ def infer_sorted_column_labels(
             )
         else:
             # self is a DataFrame, other a Series.
-            series = other.to_pandas().squeeze()
+            series_other = other.to_pandas()
```

Comment:
would this logic potentially apply to other functions as well? If so, maybe we can have a simple helper function as this is similar to LL 11010-